Synview-backend/websocket/websocket_server.ts

```diff
@@ -7,11 +7,11 @@ export async function EntrySocket(socket: WebSocket): Promise<void> {
   return;
 }
 
-export function sendQuestionToConnectedUsers(data: any) {
-  const question = JSON.stringify(data);
+export function sendDataToUsers(data: any) {
+  const jsonData = JSON.stringify(data);
   for (const socket of sockets) {
     try {
-      socket.send(question);
+      socket.send(jsonData);
     } catch {
       sockets.delete(socket);
     }
```

Comment:
nit: Would it be better to first check Role's first, to make sure you are not sending project updates to people who aren't allowed/invited to the project?